public/app/features/gops/configuration-tracker/irmHooks.ts

```diff
@@ -165,6 +175,40 @@ export function useGetEssentialsConfiguration(): EssentialsConfigurationData {
             },
             done: isCreateAlertRuleDone,
           },
+          {
+            title: 'Create your first SLO',
+            description: 'Create SLOs to monitor your service.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-slo-app/wizard/new',
+              },
+              label: 'Create',
+              urlLinkOnDone: {
+                url: '/a/grafana-slo-app/manage-slos',
+              },
+              labelOnDone: 'View',
+            },
+            done: hasSlo,
+          },
+          {
+            title: 'Enable SLO alerting',
```

Comment:
I'm not sure how to enable slo. If I click to this button it brings me to that page where I don't know what to do. What is supposed to do the user in here?

Reply:
There's not direct action to enable alerting in specific SLOs, when there's not a unique SLO with Alerting enabled, it redirects the user to Manage SLOs view and user can do it by pressing the button Enable Alerts on any SLO desired: 
<img width="193" alt="Screenshot 2024-06-18 at 10 27 49" src="https://github.com/grafana/grafana/assets/1680157/327e29d8-9faa-4f0c-861f-f501c4e8ed2a">

Reply:
maybe we should update the tooltip on the button , so users understand what to do? wdyt?